xapian-core/tests/api_weight.cc

```diff
@@ -134,6 +134,49 @@ DEFINE_TESTCASE(bm25weight5, backend) {
 }
 
 // Test exception for junk after serialised weight.
+DEFINE_TESTCASE(bm25plusweight3, !backend) {
```

Comment:
Ah, sorry I missed that. Maybe we can come up with names for them that describe what about BM25+ they're testing?